systemvm/debian/opt/cloud/bin/cs/CsLoadBalancer.py

```diff
@@ -46,7 +46,7 @@ def process(self):
             CsHelper.copy(HAPROXY_CONF_T, HAPROXY_CONF_P)
 
             proc = CsProcess(['/run/haproxy.pid'])
-            if not proc.find():
+            if proc.grep("/usr/sbin/haproxy") == -1:
```

Comment:
@ustcweizhou this is the same fix done in the PR: https://github.com/apache/cloudstack/pull/4386, changes in "_CsRedundant.py_" for keepalived process. As I've commented in that PR, on a broader view, other processes might have similar issue. can this be fixed in CsProcess.py find() ? else at some point, we can end up replacing all _proc.find()_ with _proc.grep()_.

Reply:
@sureshanaparti I will make change on proc.find